libavfilter/dnn/dnn_interface.c

```diff
@@ -33,6 +33,10 @@ DNNModule *ff_get_dnn_module(DNNBackendType backend_type)
 {
     DNNModule *dnn_module;
 
+    // temporary change till unification completes in atleast one backend
+    av_log(NULL, AV_LOG_ERROR, "Execution currently not supported \n");
```

Comment:
it's not good to disable a feature even it is temporary, we need to refine the patches

Reply:
it is a good method for close/internal code, but it is not good for open source.

Reply:
Okay, got it.
How about reordering the "Task-based Inference in Native Backend" to first and then combining the unification patches in the three backend to one?

Reply:
looks good, hope the merged patch is not big, :)

Reply:
I'll try not to make the patch too large. Once refining is complete, I'll tell you. Thank you.

Reply:
I tried merging the patches. The first two patches are a bit large, but the next two are small.